services/viva/microservice/test/helpers/attachment.test.ts

```diff
@@ -19,18 +20,30 @@ it('return a list of attachment objects', async () => {
       Body: 'Some body hereA0',
     });
 
-  const answerList = [
+  const answerList: CaseFormAnswer[] = [
     {
       field: {
         id: '123',
         tags: [],
       },
       value: [
         {
-          uploadedFileName: 'uploadedFileName_0.png',
+          uploadedId: 'uploadedId_0.png',
```

Comment:
The uploadId is actually an uuid. Doesn't really matter for the test but could be good to follow the format.